index.html

```diff
@@ -773,7 +773,7 @@
 			<li>If the pointer is not in the <a>active buttons state</a>, then terminate these steps.</li>
 			<li>For the specified <code>pointerId</code>, set the <dfn>pending pointer capture target override</dfn> to the <code>Element</code> on which this method was invoked.</li>
 		</ol>
-		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements.</div>
+		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements. The region of the capturing element includes the element itself and all its children to mimic the hover state of the capturing element. Moving out of (into) this region causes <code>pointerout</code> and <code>pointerleave</code> (<code>pointerover</code> and <code>pointerenter</code>) to be sent to the capturing element and all its ancestors.</div>
```

Comment:
It doesn't bubble. But as far as I remember Edge does send it (and target it to them if I recall correctly) to all its ancestor one by one like the normal pointerenter/leave. It is as if that as soon as you are out of the capturing element you are out of the whole document.